js/components/graph/EdgeGroup.js

```diff
@@ -29,18 +25,18 @@ export default class EdgeGroup extends React.Component {
     });
   }
 
-  reset() {
+  reset = ()=> {
     this.props.edgesJSON.forEach(edgeJSON => {
       this[edgeJSON.id_].setState({ status: "inactive" });
     });
   }
 
-  setRefEntry(edgeJSON) {
+  setRefEntry = (edgeJSON) =>{
```

Comment:
Don't include parentheses around a single parameter; that is, you can write `edgeJSON => { ...`

Reply:
Thank you! Are some of these issues supposed to be automatically fixed by me running 'yarn run prettier'?

Reply:
Hmm not exactly, although I think the pre-commit hook is supposed to detect style issues like this one. For now you can fix these manually, and I'll check out the pre-commit configuration.